### PR TITLE
SIG-3540 Document secure download feature

### PR DESCRIPTION
**Pull request by evoronkin**

### CODE REVIEW

**Review by ATimmeh33 — APPROVED** [review]
Technical review approved

File: _data/api-swagger.yaml [hunk 1]
```diff
@@ -564,6 +576,10 @@ definitions:
             Must conform to E.164,
             [the international public telecommunication numbering plan](https://en.wikipedia.org/wiki/E.164),
             which requires the country calling code (e.g. +31).
+        SecureDownload:
```
Comment (generateui) [inline _data/api-swagger.yaml:579]: specify default value

**Review by generateui — CHANGES_REQUESTED** [review]
Please rewrite that first sentence, and specify default value if possible. Otherwise looks great :)

File: _data/api-swagger.yaml [hunk 2]
```diff
@@ -543,6 +551,10 @@ definitions:
           description: |
             The level of confidence with which the identity of the signer has been determined.
             For further information, please refer to [Logius](https://www.logius.nl/diensten/digid/hoe-werkt-het).
+        SecureDownload:
+          type: boolean
+          description: |
+            Optional. Authentication method to be used to download the signed document(s).
```
[the remark this answers is not in the document]
Comment (PaulLoth) [reply _data/api-swagger.yaml:557]: I suggest: default = false.

That takes away confusion. Where customers try to interpret a postback or response with this attribute missing they can still deduce expected behaviour of transaction

File: _data/api-swagger.yaml (hunk 1 above)
Comment (PaulLoth) [reply _data/api-swagger.yaml:579]: I suggest: default = false.

That takes away confusion. Where customers try to interpret a postback or response with this attribute missing they can still deduce expected behaviour of transaction

File: _data/api-swagger.yaml [hunk 3]
```diff
@@ -275,6 +275,8 @@ definitions:
                 The order in which the authentications are provided determine in which order the signer will have to perform the specified method.
 
                 Authentications must be performed before the document(s) can be viewed.
+                If the option specified for the concrete authentication to be used for the secure download of the signed document(s)
```
[the remark this answers is not in the document]
Comment (PaulLoth) [reply _data/api-swagger.yaml:278]: "When configured, the download url for a signer is authenticated with the same method as the signing url.
The download url is returned in the response, and (optionally) emailed to the signer."

**Review by ATimmeh33 — APPROVED** [review]

**Pull request merged**